### ISSUE TITLE
Cannot add iPad screenshots

### ISSUE THREAD

**Issue by bunuelcubosoto** [chbeer/Jack-Support#52]
Hi,

I cannot add iPad screenshots from Jack. The iPad section is missing. It's probably because the current version of my application is iPhone only.

Is there a way to force iPad section?

Best,
Bunuel

**Comment by chbeer** [comment]
Can you add iPad screenshots on iTunes Connect?

Did you already upload a build with iPad support?

**Comment by bunuelcubosoto** [comment]
Yes, I can. I uploaded some TestFlight builds but I haven't yet submitted a build to App Review team.
